docs/kubectl-equivalence-in-java.md

```diff
@@ -0,0 +1,246 @@
+#Kubectl Equivalence in Java
+
+__TL;DR__: Get used to kubectl? Now our kubernetes Java client library has releases a set of 
+helpful client utilities which has the similar input argument interface as the kubectl binary.
+Especially the developers who're already familiar with kubectl commands, after reading this 
+document, you will know how to program to operate kubernetes as neat as kubectl.
+
+
+### What is Java Kubectl
+
+The Java Kubectl is not only a more user-friendly wrapper for our direct HTTP kubernetes client,
+but also contains implementation of commonly-used kubectl advanced commands. All these kubectl 
+equivalences are accessible as a group of static helper functions under `io.kubernetes.client.extended.kubectl.Kubectl` 
+class. You can import them by adding the following dependency to your project:
+
+```xml
+<dependency>
+	<groupId>io.kubernetes</groupId>
+	<artifactId>client-java-extended</artifactId>
+	<version>${latest project version}</version>
+</dependency>
+```
+
+Now you're all set, invoke the kubectl commands as Java static functions wherever you like in 
+your project.
+
+#### (Optional) Setting a Global Client-Config
+
+Kubectl static helpers doesn't know its client-config (or kubeconfig if the name makes more sense to 
+you) unless you set it when your application starts. You can either specifies the client-config upon 
+invoking kubectl helpers or simply set a global config at the start of your application:
+
+```java
+Configuration.setDefaultApiClient(ClientBuilder.defaultClient());
+```
+
```

Comment:
This will create a client on either a client, or a container in Kubernetes

Reply:
fixed